tweetflask.py

```
@@ -0,0 +1,88 @@
+from flask import Flask, jsonify, request
+from flask_basicauth import BasicAuth
+import json
+import yaml
+import ssl
+app = Flask(__name__)
+basic_auth = BasicAuth(app)
+app.config['JSON_SORT_KEYS'] = False
+app.config['BASIC_AUTH_FORCE'] = True
```

When I sent a request with an unauthorized user, I will get 401 error code but with an empty response object. The API should return an errors object to adhere to the JSON API specification as the response in this case as well. Also, can you move these config setting into `main` or some other functions? We don't want code spread everywhere in the file.